content/general/limits.textile

```diff
@@ -0,0 +1,314 @@
+---
+title: Limits
+section: general
+index: 20
+meta_description: "The limits that apply to operations on your account."
+meta_keywords: "Ably, limit, rate, maximum, quota, peak."
+jump_to:
+  Help with:
+    - Overview#overview
+    - Quota limits#quota
+    - Token limits#token
+    - Connection limits#connection
+    - Channel limits#channel
+    - Message limits#message
+    - Queue limits#queues
+    - Integration limits#integration
+    - API limits#api
+---
+
+Limits are applied to all accounts in order to prevent any impact on service caused by deliberate or accidental abuse. Limits vary depending on the "package":https://ably.com/pricing associated with your account and upgrading to a package with additional message, connection or channel allowance increases any related limits. Enterprise package limits are customizable.
+
+h2(#overview). Overview
+
+The majority of limits fall into one of three categories: "rate limits":#rates, "time-based limits":#time or "quantity-based limits":#quantity. The effect of hitting or exceeding a limit varies depending on its type. 
+
+Limits are also categorized as local or global depending on whether they relate to a single resource, or the aggregate of all resources of a given type:
+
+- Local limits := relate to a single resource, such as an individual channel or connection. Examples of local limits include the maximum number of channels per connection and the maximum publish rate on a channel. Email notifications are not sent for local limits.
+- Global limits := relate to all resources associated with the account, such as peak connections. Global limits are an aggregate of usage across all applications in your account. Email notification warnings will be sent when a global limit is close to being reached, or has been hit.
+
+h3(#rates). Rate limits
```

Comment:
I separated them as I thought it would be clearer because of the differences in the 2 rate limits.

I've merged them into just time and quantity now though as per your suggestion and it does look a lot simpler, so thanks.

Reply:
I'm sorry, but I don't agree with this change. Instantaneous rate limits are not measured per second, in the sense that they reset every second. The unit of measurement is "per second", but they need to be thought of as instantaneous rates. I suggest reverting https://github.com/ably/docs/commit/e9af3ddc3a149b181b34ed04b4ff2edbed96eee4.

Reply:
Thanks Paddy. I've reverted [e9af3dd](https://github.com/ably/docs/commit/e9af3ddc3a149b181b34ed04b4ff2edbed96eee4).